torch/lib/c10d/Store.hpp

```diff
@@ -55,6 +55,30 @@ class Store : public torch::CustomClassHolder {
 
   virtual void setTimeout(const std::chrono::milliseconds& timeout);
 
+  /*
```

Comment:
nit: comments in other parts of the code base usually use // instaed of /**/, even for multi-line comments.